skins/cat17/src/app/lib/form_validation.js

```diff
@@ -209,13 +208,15 @@ var jQuery = require( 'jquery' ),
 	/**
 	 *
 	 * @param {string} validationUrl
+	 * @param {object} amountFormatter Formatter object that supports the .format method
 	 * @param {Function} sendFunction jQuery.post function or equivalent
 	 * @return {AmountValidator}
 	 */
-	createAmountValidator = function ( validationUrl, sendFunction ) {
+	createAmountValidator = function ( validationUrl, amountFormatter, sendFunction ) {
 		return objectAssign( Object.create( AmountValidator ), {
 			validationUrl: validationUrl,
-			sendFunction: sendFunction || jQuery.post
+			sendFunction: sendFunction || jQuery.post,
+			amountFormatter: amountFormatter || { format: function( v ) { return v; } }
```

Comment:
I could not find a place where no `amountFormatter` is injected. Can we then please lose the vicious default (`||`)?